### DISCUSSION TITLE
How to change font size of Table Data in Node Red?

### DISCUSSION THREAD

**Discussion by techylax** [TotallyInformation/node-red-contrib-uibuilder#218] [Q&A]
Hello everyone,

I'm new to Node-RED and I'm currently facing a challenge with adjusting the font size in my table data. The HMI UI I'm working on lacks optimization, and I've made several attempts to resolve this issue without success.

I've experimented with different approaches, including utilizing the Function and Template nodes and incorporating CSS within them. However, I haven't achieved the desired outcome so far.

If anyone could offer assistance with this matter, I would greatly appreciate it. Any suggestions or guidance on how to effectively change the font size in my table data would be immensely helpful. Thank you for your time and support

![fffff](https://github.com/TotallyInformation/node-red-contrib-uibuilder/assets/22542035/9e583a75-9171-4555-8927-0f7c02d42f57)
![ttttttt](https://github.com/TotallyInformation/node-red-contrib-uibuilder/assets/22542035/382756b0-7b77-4b8c-ab84-b2a224907578)

**Comment by TotallyInformation** [comment]
Hi, looks like you are in the wrong place I'm afraid. You are using a Dashboard node, not uibuilder? As such, you need to raise this in the Node-RED Discourse forum.

Let me know if there is any help I can give with in relation to uibuilder.
